sdk/cosmos/ci.spark.databricks.yml

```diff
@@ -58,10 +59,10 @@ stages:
       - task: Bash@3
         displayName: Importing Jars
         inputs:
-          filePath: $(build.sourcesdirectory)/sdk/cosmos/azure-cosmos-spark_3-1_2-12/test-databricks/databricks-jar-install.sh
-          arguments: 'oltp-ci-2workers-ds3v2 $(build.sourcesdirectory)/sdk/cosmos/azure-cosmos-spark_3-1_2-12/target'
+          filePath: $(build.sourcesdirectory)/sdk/cosmos/$(SparkVersion)/test-databricks/databricks-jar-install.sh
```

Comment:
These need to be `${{ parameters.SparkVersion }}` I believe.

Reply:
Good catch!